project-management/src/main/java/life/qbic/projectmanagement/application/ContactRepository.java

```diff
@@ -0,0 +1,29 @@
+package life.qbic.projectmanagement.application;
+
+import java.util.List;
+import life.qbic.projectmanagement.domain.model.project.Contact;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ContactRepository {
+
+  public List<Contact> findAll() {
+    //TODO implement
+    return dummyContacts();
+  }
+
+  private static List<Contact> dummyContacts() {
+    return List.of(
+        new Contact("Max Mustermann", "max.mustermann@qbic.uni-tuebingen.de"),
+        new Contact("David Müller", "david.mueller@qbic.uni-tuebingen.de"),
+        new Contact("John Koch", "john.koch@qbic.uni-tuebingen.de"),
+        new Contact("Trevor Noah", "trevor.noah@qbic.uni-tuebingen.de"),
+        new Contact("Sarah Connor", "sarah.connor@qbic.uni-tuebingen.de"),
+        new Contact("Anna Bell", "anna.bell@qbic.uni-tuebingen.de"),
+        new Contact("Sophia Turner", "sophia.turner@qbic.uni-tuebingen.de"),
+        new Contact("Tylor Smith", "tylor.smith@qbic.uni-tuebingen.de")
+    );
+  }
+
+
```

Comment:
Nice test implementation, however down the line this should be replaced with a "real" database

Reply:
yes can be done separately